patches/ygot/ygot.patch

```diff
@@ -1,6 +1,58 @@
+diff -ruN ygot-dir-orig/ygot/generator/generator.go ygot-dir/ygot/generator/generator.go
+--- ygot-dir-orig/ygot/generator/generator.go	2020-06-21 15:35:13.777667000 -0700
++++ ygot-dir/ygot/generator/generator.go	2020-06-21 15:27:34.931674000 -0700
+@@ -109,7 +109,15 @@
+ }
+ 
+ // writeIfNotEmpty writes the string s to b if it has a non-zero length.
+-func writeIfNotEmpty(b io.StringWriter, s string) {
++// commented out the below method, since "io.StringWriter" is introduced in go 1.12
++//func writeIfNotEmpty(b io.StringWriter, s string) {
++//	if len(s) != 0 {
++//		b.WriteString(s)
++//	}
++//}
++
++// temp. - remove this method when we support go 1.12, and uncomment above method
++func writeIfNotEmpty(b *strings.Builder, s string) {
+ 	if len(s) != 0 {
+ 		b.WriteString(s)
+ 	}
+diff -ruN ygot-dir-orig/ygot/genutil/common.go ygot-dir/ygot/genutil/common.go
+--- ygot-dir-orig/ygot/genutil/common.go	2020-06-21 15:35:13.763550000 -0700
++++ ygot-dir/ygot/genutil/common.go	2020-06-21 15:27:34.925858000 -0700
+@@ -18,9 +18,9 @@
+ 
+ import (
+ 	"fmt"
+-	"io"
+ 	"sort"
+-
++	"strings"
++	
+ 	"github.com/openconfig/goyang/pkg/yang"
+ 	"github.com/openconfig/ygot/util"
+ 	"github.com/openconfig/ygot/ygot"
+@@ -42,7 +42,15 @@
+ )
+ 
+ // WriteIfNotEmpty writes the string s to b if it has a non-zero length.
+-func WriteIfNotEmpty(b io.StringWriter, s string) {
++// commented out the below method, since we don't support go 1.12 currently
++//func WriteIfNotEmpty(b io.StringWriter, s string) {
++//	if len(s) != 0 {
```

Comment:
Same. Please remove these commented lines.

Reply:
Addressed it. Thanks..